helpers/hasCorrectNextConfig.js

```diff
@@ -12,9 +11,27 @@ const hasCorrectNextConfig = async ({ nextConfigPath, failBuild }) => {
   const isValidTarget = acceptableTargets.includes(target)
   if (!isValidTarget) {
     console.log(
-      `Your next.config.js must set the "target" property to one of: ${acceptableTargets.join(', ')}. Update the
-      target property to allow this plugin to run.`,
+      `The "target" config property must be one of "${acceptableTargets.join('", "')}". Setting it to "serverless".`,
     )
+
+    /* eslint-disable fp/no-delete, node/no-unpublished-require */
+
+    // We emulate Vercel so that we can set target to serverless if needed
+    process.env.NOW_BUILDER = true
+    // If no valid target is set, we use an internal Next env var to force it
+    process.env.NEXT_PRIVATE_TARGET = 'serverless'
+
+    // 🐉 We need Next to recalculate "isZeitNow" var so we can set the target, but it's
+    // set as an import side effect so we need to clear the require cache first. 🐲
+    // https://github.com/vercel/next.js/blob/canary/packages/next/telemetry/ci-info.ts
+
+    delete require.cache[require.resolve('next/dist/telemetry/ci-info')]
+    delete require.cache[require.resolve('next/dist/next-server/server/config')]
+
+    // Clear memoized cache
+    getNextConfig.clear()
```

Comment:
why do we have to clear here?

Reply:
The function is memoized, so it will return the old value if we don't clear it

Reply:
This works. 
In the long run, we probably should consider removing memoization on `getNextConfig()` and passing its return value around function calls instead.